ReactAndroid/src/main/java/com/facebook/react/ReactRootView.java

```diff
@@ -200,6 +203,52 @@ public boolean onTouchEvent(MotionEvent ev) {
     return true;
   }
 
+  @Override
+  public boolean dispatchKeyEvent(KeyEvent ev) {
+    if (mReactInstanceManager == null || !mIsAttachedToInstance ||
+      mReactInstanceManager.getCurrentReactContext() == null) {
+      FLog.w(
+        ReactConstants.TAG,
+        "Unable to handle key event as the catalyst instance has not been attached");
+      return super.dispatchKeyEvent(ev);
+    }
+    mAndroidTVRootViewHelper.handleKeyEvent(ev, getDeviceEventEmitter());
+    return super.dispatchKeyEvent(ev);
+  }
+
+  @Override
+  protected void onFocusChanged(boolean gainFocus, int direction, Rect previouslyFocusedRect) {
+    if (mReactInstanceManager == null || !mIsAttachedToInstance ||
+      mReactInstanceManager.getCurrentReactContext() == null) {
+      FLog.w(
+        ReactConstants.TAG,
+        "Unable to handle focus changed event as the catalyst instance has not been attached");
+      super.onFocusChanged(gainFocus, direction, previouslyFocusedRect);
+      return;
+    }
+    mAndroidTVRootViewHelper.clearFocus(getDeviceEventEmitter());
```

Comment:
Note: I'm not sure about this but I assume that since it is the `RootView` that looses its focus that means another (native?) part of the app is now focused and we should send blur event to react-native views.